### ISSUE TITLE
getSitelinkData throwing error because of missing language

### ISSUE THREAD

**Issue by noinkling** [maxlath/wikidata-sdk#43]
```
> wdk.getSitelinkData('shnwiki')
Error: sitelink lang not found: shn
```
It's present in the list at https://www.wikidata.org/w/api.php?action=help&modules=wbgetentities and you can find an example of usage here: https://www.wikidata.org/wiki/Special:EntityData/Q1860.json

```
{"site":"shnwiki","title":"လိၵ်ႈဢင်းၵိတ်ႉ","badges":[],"url":"https://shn.wikipedia.org/wiki/%E1%80%9C%E1%80%AD%E1%81%B5%E1%80%BA%E1%82%88%E1%80%A2%E1%80%84%E1%80%BA%E1%80%B8%E1%81%B5%E1%80%AD%E1%80%90%E1%80%BA%E1%82%89"}
```

I don't know if any other languages are missing.
@maxlath Do you generate the array in [sitelinks_languages.js](https://github.com/maxlath/wikidata-sdk/blob/master/lib/helpers/sitelinks_languages.js) with a script or is it done by hand?

**Comment by maxlath** [comment]
@noinkling I think I did it by hand but I'm looking at a way to write a script to easily update it: scrapping https://www.wikidata.org/w/api.php?action=help&modules=wbgetentities could be an option, but I would prefer getting the list of sites keys from a JSON file somewhere, any clue where to find it?

**Comment by noinkling** [comment]
@maxlath I had a look at the network requests for the API sandbox to see how it gets them, looks like it uses the `paraminfo` action, i.e:

https://www.wikidata.org/w/api.php?action=paraminfo&modules=wbgetentities

**Comment by maxlath** [comment]
done! patch published in v5.15.11
